Documentation/building/linux-instructions.md

```diff
@@ -37,7 +40,7 @@ ellismg@linux:~$ sudo apt-get update
 
 Then install the packages you need:
 
-`ellismg@linux:~$ sudo apt-get install cmake llvm-3.5 clang-3.5 lldb-3.6 lldb-3.6-dev libunwind8 libunwind8-dev gettext libicu-dev`
+`ellismg@linux:~$ sudo apt-get install cmake llvm-3.5 clang-3.5 lldb-3.6 lldb-3.6-dev libunwind8 libunwind8-dev gettext libicu-dev curl libcurl4-openssl-dev libssl-dev`
```

Comment:
I don't think `curl` should be considered as dependency. The script checks for both `wget` (for Debian-like OSes) and `curl` (rest of the POSIX world) and then selects the command which is available. I assume XUbuntu must have either wget or curl command pre-installed? it would be very weird if neither of the commands are available .. 

Alternatively, if we are making `curl` as a "hard" / set-in-stone dependency in build instructions, then this part of script should be simplified: https://github.com/dotnet/coreclr/blob/5aedb627a91/build.sh#L109-L120 to:

```bash
echo "Restoring NuGet.exe..."
curl -sSL --create-dirs -o $__NuGetPath https://api.nuget.org/downloads/nuget.exe
```
without defensive error handling, in order to encourage users [RTFM](https://en.wikipedia.org/wiki/RTFM).